lib/ansiblelint/__main__.py

```diff
@@ -29,20 +29,21 @@
 from ansiblelint import cli, default_rulesdir, RulesCollection, Runner
 from ansiblelint.utils import get_playbooks_and_roles
 from ansiblelint.utils import normpath, initialize_logger
+from typing import Set, Any
 
 
 _logger = logging.getLogger(__name__)
 
 
-def main():
+def main() -> int:
     cwd = pathlib.Path.cwd()
 
     options = cli.get_config(sys.argv[1:])
 
     initialize_logger(options.verbosity)
     _logger.debug("Options: %s", options)
 
-    formatter_factory = formatters.Formatter
+    formatter_factory: Any = formatters.Formatter
```

Comment:
Why is it any?

Reply:
That is a hack to make mypy happy about allowing other classes to be assigned as formatters. Otherwise, it will complain on assignment. I was not able to find another workaround yet. Is one of the known issues with python logging and mypy. Even the stdlib has only partial stubs.